cmd/mr_show.go

```diff
@@ -239,9 +240,9 @@ func printMR(mr *gitlab.MergeRequest, project string, renderMarkdown bool) {
 
 func init() {
 	mrShowCmd.Flags().BoolP("no-markdown", "M", false, "don't use markdown renderer to print the issue description")
-	mrShowCmd.Flags().BoolP("comments", "c", false, "show comments for the merge request")
+	mrShowCmd.Flags().BoolP("comments", "c", false, "show comments for the merge request (does not work with --patch)")
 	mrShowCmd.Flags().StringP("since", "s", "", "show comments since specified date (format: 2020-08-21 14:57:46.808 +0000 UTC)")
-	mrShowCmd.Flags().BoolVarP(&mrShowPatch, "patch", "p", false, "show MR patches")
+	mrShowCmd.Flags().BoolVarP(&mrShowPatch, "patch", "p", false, "show MR patches (does not work with --comments)")
```

Comment:
What if: instead of warning it in the help message, you actually block it in code level, returning error if both are set.
Like we do for `lab mr edit|create` with `-F` and `-m` being used together.

Reply:
Good idea :) but that doesn't work because if I set comments = true in the .config, then "show -p" would fail 100% of the time.

Reply:
Fair point.
Would that make sense to print an WARNING at least?
In the help message it seems pretty easy to miss, that's why I'm asking for some runtime feedback from lab to the user :)

Reply:
I'd still end up seeing the warning everytime I ran it :(

Reply:
Well.. ok then :)
I'm fine with it.